### PR TITLE
Cr/fix

### PR DESCRIPTION
**Pull request by Tzurrr**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (nadav5200) [inline .idea/.gitignore:3]: add to the git ignore file to not upload the .idea files

File: elogger.py [hunk 2]
```diff
@@ -1,26 +1,23 @@
...
 import json_parser
 
+
 def write_logs_to_elastic(event_string):
-    conf_dict = json_parser.parse_json_to_var("/home/tzur/client/config.json")
+    conf_dict = json_parser.parse_json_to_var("/home/tzur/server/config.json")
     url_path = conf_dict["kibanas_url"]
-    logger = logging.getLogger("app")
+
+    if url_path == "":
```
Comment (nadav5200) [inline elogger.py:11]: is this line necessary?

**Review by nadav5200 — CHANGES_REQUESTED** [review]

File: .idea/.gitignore (hunk 1 above)
Comment (Tzurrr) [reply .idea/.gitignore:3]: V

File: elogger.py (hunk 2 above)
Comment (Tzurrr) [reply elogger.py:11]: V

**Pull request merged**